src/cli/project/version/get.rs

```diff
@@ -0,0 +1,14 @@
+use crate::Project;
+use clap::Parser;
+
+/// Get the project version.
+#[derive(Parser, Debug, Default)]
+pub struct Args {}
```

Comment:
Even when empty I kept all the `Args`. Should I get rid of them?

Reply:
Yep just get rid of them.

Reply:
I did and also refactored the command docstring into `mod.rs`.